arenadata/Dockerfile

```diff
@@ -75,6 +75,11 @@ COPY . gpdb_src
 RUN rm -rf gpdb_src/.git/
 
 FROM base as test
+# Reinstall glibc-common. This is necessary to get langpacks in docker
```

Comment:
some notice for [lines](https://github.com/arenadata/gpdb/pull/481/files#diff-148eeb204cd18bca72d96990f7c3842fedbc9f03051f2e9758ef257c03547f45R37-R39): https://stackoverflow.com/a/27504514/4139676

Reply:
I don't understand how this is connected to my patch